### DISCUSSION TITLE
v0.10.3

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#880] [Releases]
https://github.com/AsamK/signal-cli/blob/master/CHANGELOG.md#0103---2022-02-01

**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.10.2...v0.10.3

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.3'>v0.10.3</a>.</em>

**Comment by elXaptation** [comment]
Hi downloaded signal-cli v0.10.3, following the [Quick Start](https://github.com/AsamK/signal-cli/wiki/Quickstart) I'm getting the missing native library dependency error for libsignal-client. Am I understanding [signal-libs-build](https://github.com/exquo/signal-libs-build) correctly that libsignal-client for the x86_64 are bundled in to the pre-built signal-client? Or do I still need to download the files from [libsignal-client/releases](https://github.com/signalapp/libsignal-client/releases/) and put them in the right place? Thank you.

**Comment by exquo** [comment]
Yes, the latest signal-cli releases should work out of the box.
What is your OS?

**Reply by elXaptation** [reply]
Thank you for responding.

PROMPT:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.6 LTS
Release:	18.04
Codename:	bionic

I downloaded the v0.12.2  libsignal_jni.so  file and bundled in to the .jar file and also copied it to my java.library.path. Still getting the error with the register operation.

**Reply by exquo** [reply]
The issue is an older `glibc` version: see #643. You can download the [pre-built file](https://github.com/exquo/signal-libs-build/releases/download/libsignal-client_v0.11.0/libsignal_jni.so-v0.11.0-x86_64-unknown-linux-gnu.tar.gz) and repackage according to the instructions.

**Reply by elXaptation** [reply]
Closing the loop. The prebuilt file is working great. my signal-cli is up and running on Ubuntu 18.04.06 LTS.

**Comment by elXaptation** [comment]
Cool. Thank you. I'll give it a try.
